OpenRA.Game/Traits/World/ScreenMap.cs

```diff
@@ -33,6 +33,7 @@ public class ScreenMap : IWorldLoaded
 		readonly Cache<Player, SpatiallyPartitioned<FrozenActor>> partitionedFrozenActors;
 		readonly SpatiallyPartitioned<Actor> partitionedActors;
 		WorldRenderer worldRenderer;
+		public Viewport Viewport { get; private set; }
```

Comment:
This avoids explicitly creating an instance member variable. However, if you prefer:
```c#
public Viewport Viewport { get { return viewport; } }
Viewport viewport;
```
I could easily change it.